src/shared/route-urls.ts

```diff
@@ -26,8 +26,9 @@ export enum RouteUrls {
   EditNetwork = '/edit-network',
   Fund = '/fund/:currency',
   FundChooseCurrency = '/fund-choose-currency',
-  IncreaseStxFee = '/increase-fee/stx/:txid',
+  IncreaseStacksFee = '/increase-fee/stacks/:txid',
```

Comment:
I guess `/increase-fee/btc` should also update here?

Reply:
I just saw you changed:
- `IncreaseStxFee` -> `IncreaseStacksFee` 
- `'/increase-fee/stx/:txid'` -> `/increase-fee/stacks/:txid`

Should we also have `IncreaseBitcoinFee`  & ``/increase-fee/bitcoin/:txid` then?

Reply:
OK 👍 I was just suggesting for consistency as there is a mix of conventions here